.github/ci_templates/helper_jobs.yml

```diff
@@ -80,15 +89,21 @@ git_info:
             githubBranch  = GITHUB_REF_TYPE == "branch" ? refName : ""
             githubSHA     = context.sha
             core.info(`${context.eventName} event: set git info from context: ${JSON.stringify({GITHUB_REF_NAME, GITHUB_REF_TYPE, sha: context.sha })}`)
+            if (context.issue && context.issue.number) {
+              prNumber = context.issue.number
+            }
```

Comment:
This check is useless for other triggers (workflow_dispatch without pull_request_ref, schedule, push).